### PR TITLE
final update pl_pl.lang

### PR DESCRIPTION
**Pull request by EliogabalusPL**
Fixed 3 variables that changed in the process of creating new module versions. Fixed an invalid name. The names of cables and pipes are correct for the Polish language.

### CODE REVIEW

File: src/main/resources/assets/mekanism/lang/pl_pl.lang [hunk 1]
```diff
@@ -187,31 +187,31 @@ tile.OreBlock.CopperOre.name=Ruda miedzi
...
-tile.Transmitter.DiversionTransporter.name=Transporter różnicujący
-tile.Transmitter.BasicThermodynamicConductor.name=Podstawowy przewodnik termodynamiczny
-tile.Transmitter.AdvancedThermodynamicConductor.name=Zaawansowany przewodnik termodynamiczny
-tile.Transmitter.EliteThermodynamicConductor.name=Profesjonalny przewodnik termodynamiczny
-tile.Transmitter.UltimateThermodynamicConductor.name=Najlepszy przewodnik termodynamiczny
+tile.Transmitter.BasicPressurizedTube.name=Podstawowy gazociąg
+tile.Transmitter.AdvancedPressurizedTube.name=Zaawansowany gazociąg
+tile.Transmitter.ElitePressurizedTube.name=Profesjonalny gazociąg
+tile.Transmitter.UltimatePressurizedTube.name=Najlepszy gazociąg
+tile.Transmitter.BasicUniversalCable.name=Podstawowy kabel energetyczny
```
Comment (thiakil) [inline src/main/resources/assets/mekanism/lang/pl_pl.lang]: Why removing 'universal' from the name?

File: src/main/resources/assets/mekanism/lang/pl_pl.lang [hunk 2]
```diff
@@ -972,6 +972,7 @@ itemGroup.tabMekanism=Mekanism
 
 //NEI stuff
 nei.chemicalInjectionChamber=Komora wtrysku chemicz.
+nei.rotaryCondensentrator=Kondensentr. rotac.
```
Comment (thommy101) [inline src/main/resources/assets/mekanism/lang/pl_pl.lang]: This one is removed in the 1.12 branch.

File: src/main/resources/assets/mekanism/lang/pl_pl.lang [hunk 3]
```diff
@@ -1080,28 +1081,28 @@ item.BronzeHoe.name=Motyka z brązu
...
-item.GlowstoneChestplate.name=Jasnogłazowy napierśnik
-item.GlowstoneLeggings.name=Jasnogłazowe nogawice
-item.GlowstoneBoots.name=Jasnogłazowe buty
 item.GlowstonePaxel.name=Jasnogłazowy łopasiekilof
 item.GlowstonePickaxe.name=Jasnogłazowy kilof
 item.GlowstoneAxe.name=Jasnogłazowa siekiera
 item.GlowstoneShovel.name=Jasnogłazowa łopata
 item.GlowstoneHoe.name=Jasnogłazowa motyka
 item.GlowstoneSword.name=Jasnogłazowy miecz
+item.GlowstoneHelmet.name=Jasnogłazowy hełm
```
Comment (thommy101) [inline src/main/resources/assets/mekanism/lang/pl_pl.lang]: Please leave the previous order, that way it's the same as en_us.

File: src/main/resources/assets/mekanism/lang/pl_pl.lang [hunk 4]
```diff
@@ -1080,28 +1081,28 @@ item.BronzeHoe.name=Motyka z brązu
...
-item.SteelChestplate.name=Stalowy napierśnik
-item.SteelLeggings.name=Stalowe nogawice
-item.SteelBoots.name=Stalowe buty
 item.SteelPaxel.name=Stalowy łopasiekilof
 item.SteelPickaxe.name=Stalowy kilof
 item.SteelAxe.name=Stalowa siekiera
 item.SteelShovel.name=Stalowa łopata
 item.SteelHoe.name=Stalowa motyka
 item.SteelSword.name=Stalowy miecz
+item.SteelHelmet.name=Stalowy hełm
```
Comment (thommy101) [inline src/main/resources/assets/mekanism/lang/pl_pl.lang]: Please leave the previous order, that way it's the same as en_us.

**Comment by thiakil** [comment]
While you're at it, please add these new translations: https://github.com/aidancbrady/Mekanism/commit/2f8f71862c2167ba7f3c09d10d396d23fbb7d494#diff-cbf6ee77a88a174288939fc6fac40851

**Pull request merged**